packages/frontend-api/src/Model/Product/Filter/ProductFilterOptionsFactory.php

```diff
@@ -109,6 +109,24 @@ public function createProductFilterOptions(
         return $productFilterOptions;
     }
 
+    /**
+     * @param \Shopsys\FrameworkBundle\Model\Product\Filter\ProductFilterConfig $productFilterConfig
+     * @param \Shopsys\FrameworkBundle\Model\Product\Filter\ProductFilterCountData $productFilterCountData
+     * @param \Shopsys\FrameworkBundle\Model\Product\Filter\ProductFilterData $productFilterData
+     * @return \Shopsys\FrontendApiBundle\Model\Product\Filter\ProductFilterOptions
+     */
+    public function createFullProductFilterOptions(
+        ProductFilterConfig $productFilterConfig,
+        ProductFilterCountData $productFilterCountData,
+        ProductFilterData $productFilterData,
+    ): ProductFilterOptions {
+        $productFilterOptions = $this->createProductFilterOptions($productFilterConfig, $productFilterCountData, $productFilterData);
+        $this->fillBrands($productFilterOptions, $productFilterConfig, $productFilterCountData, $productFilterData);
+        self::fillParameters($productFilterOptions, $productFilterConfig, $productFilterCountData, $productFilterData);
```

Comment:
```suggestion
        $this->fillParameters($productFilterOptions, $productFilterConfig, $productFilterCountData, $productFilterData);
```

`fillParameters` is not static :wink:

Reply:
Also, I see the `fillParameters` method is overridden in `project-base` where `$category` parameter is required :sob: